python/strings_udf/cpp/CMakeLists.txt

```diff
@@ -19,9 +19,7 @@ include(rapids-cpm)
 include(rapids-cuda)
 include(rapids-find)
 
-rapids_cpm_init()
-
-rapids_cuda_init_architectures(STRINGS_UDF)
+rapids_cuda_init_architectures(strings-udf-cpp)
```

Comment:
@robertmaynard probably OK to leave, but isn't this redundant? We have no way (and no reason) to build the C++ components of strings_udf except via the Python build.

Reply:
It was unclear to me if we had a use case for building the  c++ components without python such as for node. So I erred on the side of safety and kept this logic.